### DISCUSSION TITLE
How to turn off the popup/dropdown menus in the Paris theme?

### DISCUSSION THREAD

**Discussion by pawell0** [datenstrom/community#900] [Ask a question] [answered]
Is there any quick setting to turn the popup (dropdown) menus off in the Paris theme?

**Comment by pawell0** [comment]
@annaesvensson can you help me ...?

**Comment by PetersOtto** [comment, marked as answer]
You could use the standard (stockholm) navigation.
Delete the code in the `paris-navigation.html` and insert the code from the `navigation.html`.

**Reply by pawell0** [reply]
Hi! Thanks for answering my post! I tried it and it works! Thank you.
